packages/client/src/index.ts

```diff
@@ -19,9 +19,28 @@
 
 import { DefaultRoutingV1HttpApiClient } from './client.js'
 import type { AbortOptions } from '@libp2p/interface'
-import type { PeerInfo } from '@libp2p/interface/peer-info'
+import type { PeerId } from '@libp2p/interface/peer-id'
+import type { Multiaddr } from '@multiformats/multiaddr'
+import type { IPNSRecord } from 'ipns'
 import type { CID } from 'multiformats/cid'
 
+export interface PeerRecord {
+  Schema: string
+  ID: PeerId
+  Addrs: Multiaddr[]
+  Protocols: string[]
+}
+
+// Deprecated: please use PeerRecord instead.
```

Comment:
Please use JSDoc comments with the `@deprectated` tag so the message appears in intellisense tool tips - the `//` style comment will not appear anywhere.

Reply:
Also, how "deprecated" is this type?  From what I can see it's returned from remote servers so the user cannot "use PeerRecord instead" as they have no choice in what's returned.

Reply:
@achingbrain you're right. I will undeprecate it. It is deprecated in the sense that servers should no longer send it, which they unfortunately still do.